src/App.css

```diff
@@ -1,38 +1,522 @@
-.App {
-  text-align: center;
+
```

Comment:
It's important that your design resembles as much as possible to Nelson's design 🧐 Please be sure to make the necessary changes in your CSS to solve this issue!

Your design 
![image](https://user-images.githubusercontent.com/15807343/181838760-b253cb93-f71d-432a-a2c9-c5ab524364d2.png)

- Things you could improve in your design🤓:
  - Display coins in a grid as shown in the given design (Two coins each row) 
  - Use the same color palette 
  - Follow the general layout of texts and sizes

![image](https://user-images.githubusercontent.com/15807343/181839790-3c24f07a-53d1-4c27-959d-c19cf28b0c02.png)